NGitLab/IIssueClient.cs

```diff
@@ -190,7 +190,15 @@ public interface IIssueClient
     IEnumerable<Participant> GetParticipants(ProjectId projectId, long issueIid);
 
     /// <summary>
-    /// Unsubscribe to the issue
+    /// Subscribe to the issue.
```

Comment:
When documenting a method, use the 3rd person (*descriptive form*, such as `gets`) instead of the 2nd person (*prescriptive form*, such as `get`).

This is a widespread standard, in many languages and in various environments. For example, refer to the following:
- [An example from Microsoft doc](https://learn.microsoft.com/en-us/dotnet/api/system.io.file.getattributes?view=net-9.0#system-io-file-getattributes(microsoft-win32-safehandles-safefilehandle))
- [Google developer documentation style guide](https://developers.google.com/style/reference-verbs)

```suggestion
    /// Subscribes the authenticated user to an issue to receive notifications.
```

Reply:
Applied.